src/client/components/Checker.tsx

```diff
@@ -22,17 +31,35 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
     }
   }
 
+  const evaluateOperation = (operation: Operation) => {
+    const { id, type, expression } = operation
+
+    switch (type) {
+      case 'ARITHMETIC':
+        return (variables[id] = OpArithmetic(expression, variables))
+      // TO-DO
+      // ====
+      // case 'IFELSE':
+      // case 'SWITCH':
+    }
+  }
+
   // TODO: For checkbox and radio since the value is the option index, we need to retrieve the actual value.
-  const onSubmit = (data: any) => console.log(data)
+  const onSubmit = (data: any) => {
+    // Define the input data variables
+    variables = data
+    operations.map(evaluateOperation)
```

Comment:
Maybe declare variables with `useState` and update it this way:

```suggestion
    const variables = operations.reduce(evaluateOperation, data)
    setVariables(variables)
```

Issue with the current way of declaring `variables` is that it gets reinitialised every time there's a re-render and we will lose the state. Will be an issue when we update the UI to display the results later on.